app/controllers/api_resource_evaluations_controller.rb

```diff
@@ -0,0 +1,13 @@
+class ApiResourceEvaluationsController < ApiApplicationController
+  def update
+    evaluation = params['evaluation']
+    resource_id = params['resource_id']
+    Resource.find(resource_id)
+    resource_evaluation = ResourceEvaluation
+                          .find_or_initialize_by(user_id: current_user.id,
+                                                 resource_id:)
+    resource_evaluation.evaluation = evaluation
+    resource_evaluation.save!
```

Comment:
Due to the restriction that evaluation in (1..5), if the `evaluation` parameter provided in the body is out of range (for example 0), the endpoint returns a 500 error rendered in html. I understand you will implement the error handling in another PR, so have this into account when implementing.